src/blake2b/blake2b64.ts

```diff
@@ -153,7 +223,7 @@ export function blake2bInit(outlen: u32, key: Uint8Array | null): Context {
 // Updates a BLAKE2b streaming hash
 // Requires hash context and Uint8Array (byte array)
 export function blake2bUpdate(ctx: Context, input: Uint8Array): void {
-  for (let i = 0; i < input.length; i++) {
+  for (let i = 0, len = input.length; i < len; i++) {
```

Comment:
Does something like this actually matter? I would expect the compiler to pick this up, or is it not that smart?

Reply:
Yes, it's little bit matter. AS don't do such high-level CSE transform. So better cache `len` manually for now. But it' really small improvment